alphalens/plotting.py

```diff
@@ -709,7 +721,15 @@ def plot_cumulative_returns_by_quantile(quantile_returns, ax=None):
         f, ax = plt.subplots(1, 1, figsize=(18, 6))
 
     ret_wide = quantile_returns.reset_index()\
-        .pivot(index='date', columns='quantile', values=1)
+        .pivot(index='date', columns='quantile', values=period)
+
+    if period > 1:
+        daily_returns = lambda ret, period: (
+            np.nanmean(ret) ** (1. / period)) - 1
+        period_ret_wide = ret_wide.add(1)**period
+        ret_wide = period_ret_wide.rolling(
```

Comment:
Is this way of doing rolling operations pandas 16 compatible? I thought this was just recently introduced.